.travis.yml

```diff
@@ -4,12 +4,14 @@ python:
   - "3.4"
   - "3.5"
   - "3.6"
+matrix:
+  include:
+  - python: "3.6"
+    env: TEST_SUITE=linter
 install:
   - pip install pycodestyle==2.3.1
   - pip install ./zulip
   - pip install ./zulip_bots
   - pip install ./zulip_botserver
 script:
-  - tools/lint
-  - python -m unittest discover -v zulip_botserver
-  - tools/test-bots
+  - tools/travis_tests
```

Comment:
I think a cleaner model would be to have a separate script for each, sorta like we have in the zulip/zulip project, but this is probably fine when things are this simple.